app/Models/Logo.php

```diff
@@ -43,15 +43,15 @@ class Logo extends Model
 
     protected $guarded = [];
 
-    public function user(){
+    public function user() {
```

Comment:
__[code_sniffer]__
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->